### PR TITLE
Added HTML5 challenge to portfolio and addressed some sizing issues

### PR DESCRIPTION
**Pull request by liamlwrnc**

### CODE REVIEW

**Comment by maxwell-01** [comment]
here

**Comment by maxwell-01** [comment]
Some screen resizing issues, see below:
image
image
image

**Comment by maxwell-01** [comment]
Quite a lot of empty space below contact on a full size screen, might want to reduce that or centre the content?

image

**Comment by maxwell-01** [comment]
It's generally better to use index.html for your home page (it has server side implications that we haven't learnt about on the course yet)

**Comment by maxwell-01** [comment]
It's good practice to set the document type and language at the start of an html file

**Comment by maxwell-01** [comment]
on line 10 of html file - there's an extra space before value=

**Comment by maxwell-01** [comment]
line 7 - the header element is designed to be used in the body, you should use it instead of your first section, it's not correct to put the header element inside the head of the document (also here it's not adding anything anyway)

**Comment by maxwell-01** [comment]
line 16 - once you're using header inside the body you do not need a class here as you can just select with `header`

**Comment by maxwell-01** [comment]
line 18 & 19 - you do not need a class on your h1 or h2 here, you should only have one `h1` element so can just select using that, for your h2, use `header h2`

**Comment by maxwell-01** [comment]
It's good practice to include the main body of your page (eg about, portfolio and contact sections) inside `main` section type. So then your page has `header``main``footer` - it's good for SEO

**Comment by maxwell-01** [comment]
line 16 - you could be more semantic with the ID of this section, for example intro-section. Top of page is not describing what it _does_ only _where it is_

File: project1.css [hunk 1]
```diff
@@ -43,6 +43,7 @@
 
 .intro-text {
 	text-align: center;
+	padding-top: 100px;
```
Comment (maxwell-01) [inline project1.css:45]: its better practice to add padding to the top of the section, not the intro text

**Comment by maxwell-01** [comment]
line 17, if you put your padding on your section instead of the div (better) then you do not need a class on this div, apply your styles with `.intro-section div`

**Comment by maxwell-01** [comment]
line 27, you do not need a class here, select it with `.about-section div` and you can select the h2 with `.about section h2`

**Comment by maxwell-01** [comment]
line 56 - again no class needed on the h2 element, select using `.portfolio-section h2`

**Comment by maxwell-01** [comment]
line 58 - drop your `<a>` element onto a new line and indent

**Comment by maxwell-01** [comment]
line 67 + others... you've got 3 elements with class `project-descriptors` but not corresponding CSS, best to remove unused classes

**Comment by maxwell-01** [comment]
line 72 - drop your a link onto a new line and indent

**Comment by maxwell-01** [comment]
line 82 - no need for class on the h2, select with `.contact-section h2`

**Review by maxwell-01 — CHANGES_REQUESTED** [review]
please see comments in the PR

**Comment by maxwell-01** [comment]
You're missing a site title, add to top of html file

**Comment by maxwell-01** [comment]
You still have blue lines under your footer links, use this to solve:
```
a {
    text-decoration: none;
}
```

image

File: project1.css [hunk 2]
```diff
@@ -38,69 +38,73 @@
...
 }
 
-.intro1 {
+.intro-section img {
+	position: absolute;
+	bottom: 0;
+	left: 10px;
+	height: 30vh;
```
Comment (maxwell-01) [inline project1.css]: I think the result is better with heigh: 30vw

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="project1.css">
+	<title>MyPortfolio</title>
```
Comment (maxwell-01) [inline index.html:7]: you can put a space between my and portfolio

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+</head>
+<body>
+	<header>
+		
+		<nav class="navbar">
```
Comment (maxwell-01) [inline index.html]: you only have one nav so this doesnt need a class, select with `nav {}`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+			<a href="#about" value="About">About</a>
+			<a href="#portfolio" value="Portfolio">Portfolio</a>
```
Comment (maxwell-01) [inline index.html]: im not sure the value does anything here? best to take it out

File: index.html [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+			<a href="#contact">Contact</a>
+			<a href="LiamLawrence.pdf" download>Download CV</a>
+		</nav>
+		
+	<section id="top-of-the-page" class="intro-section">
```
Comment (maxwell-01) [inline index.html]: youd be better calling the id something more semantic, eg 'intro-section'

File: index.html [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+		<div class="intro-body">
```
Comment (maxwell-01) [inline index.html:18]: you only have one div here so could select with `.intro section div` to save setting a new class, but its not scalable

File: index.html [hunk 8]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="project1.css">
+	<title>MyPortfolio</title>
+</head>
+<body>
+	<header>
```
Comment (maxwell-01) [inline index.html]: your `header` tag should come after your nav (looking at your site)

File: index.html (hunk 6 above)
Comment (maxwell-01) [inline index.html]: this section should be `header` and not just a section. The `header` is generally the section that contains `h1`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+	</div>
+	<div class="about-image">
+		<img src="meandpup.png" alt="meandmydog">
+	</div>
+		
+	</section>
+	
+	<section class="portfolio-section">
+		
+		<h2 id="portfolio">Portfolio</h2>
```
Comment (maxwell-01) [inline index.html]: you should put the id for your navigation on the section not the heading

File: index.html [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+ 			</div>
+ 			<div>
+ 				<p>As above, so below. This time it was the CSS3 logo we were tasked with. Line 'em up and I knock 'em down. </p>
+ 			</div>
+		</div>
+	</section>
+		
+	
+	<section class="contact-section">
+		<h2 id="contact">Contact</h2>
```
Comment (maxwell-01) [inline index.html]: put the id in the section element

File: index.html [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+		<p>Feel free to get in touch through any of the below</p>
+		<a href="mailto:liamlwrnce@gmail.com"><img src="emailicon.png"></a>
+		<a href="https://instagram.com/liamlwrnc"><img src="instaicon.png"></a>
+		<a href="https://www.linkedin.com/in/liam-l-08a6b3140"><img src="linkedinicon.png"></a>
+		<a href="https://github.com/liamlwrnc"><img src="githubicon.png"></a>
+		<a href="https://twitter.com/lwrnccodes"><img src="twittericon.png"></a>
+	</section>
+</main>
+ 		<a href="#top-of-the-page" class="scroll-to-top-button" title="Go to top">Top</a> 	
+ 	<footer>
```
Comment (maxwell-01) [inline index.html]: footer doesnt need this indentation

File: index.html [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+		<img src="meandpup.png" alt="meandmydog">
+	</div>
+		
+	</section>
+	
+	<section class="portfolio-section">
+		
+		<h2 id="portfolio">Portfolio</h2>
+		<div class="project-containers">
+			<div class="project-images"><a href="https://codepen.io/Brassic/pen/RwZgrRa"><img src="codepenicon.png"></a>
```
Comment (maxwell-01) [inline index.html:58]: you should an alt to your images

File: project1.css [hunk 13]
```diff
@@ -38,69 +38,73 @@
 	background-color: #fc7b03;
 	background-image: linear-gradient(to bottom right, #fc7b03, #000000);
 	position: relative;
-	height: 100vh;
```
Comment (maxwell-01) [inline project1.css:41]: check line 10 above - `align` is not an attribute

File: project1.css [hunk 14]
```diff
@@ -192,7 +199,14 @@ body {
 		gap: 20px;		
 	}
 	.navbar {
-justify-content: start;	
+		justify-content: start;
+	}
+	.navbar a {
+		width: 25%;
+	}
+	.intro-image img {
```
Comment (maxwell-01) [inline project1.css]: this code is not being run - what was the intension?

File: project1.css [hunk 15]
```diff
@@ -192,7 +199,14 @@ body {
...
+		height: 100px;
+		left: 20px;
 	}
 	.about-text {
```
Comment (maxwell-01) [inline project1.css:202]: this code is also not being used

**Review by maxwell-01 — CHANGES_REQUESTED** [review]

**Review by RichVelo — APPROVED** [review]
looks good yo

**Pull request merged**